packages/next/src/rsc/data/seo.ts

```diff
@@ -17,9 +17,8 @@ const { all: merge } = deepmerge;
 
 function isPostEntity(data: Entity | QueriedObject): data is PostEntity {
 	return (
-		typeof (data as PostEntity).yoast_head !== 'undefined' &&
-		typeof (data as PostEntity).yoast_head_json !== 'undefined' &&
```

Comment:
Is this removal intentional?

Reply:
Yes! I want this to work even without Yoast, so I should not rely on it for detecting post entity object. In the future I also want to support other plugins.